selfdrive/ui/translations/main_th.ts

```diff
@@ -0,0 +1,1303 @@
+    </message>
+    <message>
+        <location filename="../qt/maps/map_settings.cc" line="107"/>
+        <source>Get turn-by-turn directions displayed and more with a comma
+prime subscription. Sign up now: https://connect.comma.ai</source>
+        <translation>รับการแสดงเส้นทางแบบเลี้ยวต่อเลี้ยว และอื่นๆ ด้วยการสมัครบริการ
+comma prime สมัครเลย: https://connect.comma.ai</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map_settings.cc" line="164"/>
+        <source>No home
+location set</source>
+        <translation>ยังไม่ได้กำหนด
+ตำแหน่งของบ้าน</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map_settings.cc" line="169"/>
+        <source>No work
+location set</source>
+        <translation>ยังไม่ได้กำหนด
+ตำแหน่งของที่ทำงาน</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map_settings.cc" line="282"/>
+        <source>no recent destinations</source>
+        <translation>ไม่พบปลายทางล่าสุด</translation>
+    </message>
+</context>
+<context>
+    <name>MapWindow</name>
+    <message>
+        <location filename="../qt/maps/map.cc" line="178"/>
+        <source>Map Loading</source>
+        <translation>กำลังโหลดแผนที่</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map.cc" line="195"/>
+        <source>Waiting for GPS</source>
+        <translation>กำลังรอสัญญาณ GPS</translation>
+    </message>
+</context>
+<context>
+    <name>MultiOptionDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="300"/>
+        <source>Select</source>
+        <translation>เลือก</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="333"/>
+        <source>Cancel</source>
+        <translation>ยกเลิก</translation>
+    </message>
+</context>
+<context>
+    <name>Networking</name>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="30"/>
+        <source>Advanced</source>
+        <translation>ขั้นสูง</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="87"/>
+        <source>Enter password</source>
+        <translation>ใส่รหัสผ่าน</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="87"/>
+        <location filename="../qt/offroad/networking.cc" line="97"/>
+        <source>for &quot;%1&quot;</source>
+        <translation>สำหรับ &quot;%1&quot;</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="97"/>
+        <source>Wrong password</source>
+        <translation>รหัสผ่านผิด</translation>
+    </message>
+</context>
+<context>
+    <name>NvgWindow</name>
+    <message>
+        <location filename="../qt/onroad.cc" line="218"/>
+        <source>km/h</source>
+        <translation>ก.ม./ช.ม.</translation>
```

Comment:
Not sure, is `km/h` universally recognizable, or is the translation more recognizable?